src/issues/issue-1/sext-ed.js

```diff
@@ -0,0 +1,417 @@
+
+  const guidelines = data.allFile.edges.find(
+    edge => edge.node.name === "guidelines"
+  )
+
+  const phone = data.allFile.edges.find(edge => edge.node.name === "phone")
+
+  const communicationDesktop = data.allFile.edges.find(
+    edge => edge.node.name === "2-good-communication"
+  )
+  const communicationMobile = data.allFile.edges.find(
+    edge => edge.node.name === "2-good-communication-mobile"
+  )
+  const displayGuideline2 = isPhoneWide
+    ? communicationMobile
+    : communicationDesktop
+
+  const listen = data.allFile.edges.find(edge => edge.node.name === "listen")
+
+  const dirtyTalkDesktop = data.allFile.edges.find(
+    edge => edge.node.name === "3-dirty-talk"
+  )
+  const dirtyTalkMobile = data.allFile.edges.find(
+    edge => edge.node.name === "3-dirty-talk-mobile"
+  )
+  const displayGuideline3 = isPhoneWide ? dirtyTalkMobile : dirtyTalkDesktop
+
+  const dirtyTalk2 = data.allFile.edges.find(
+    edge => edge.node.name === "dirty-talk-2"
+  )
+
+  const sexShakespeareDeskTop = data.allFile.edges.find(
+    edge => edge.node.name === "4-sex-shakespeare"
+  )
+  const sexShakespeareMobile = data.allFile.edges.find(
+    edge => edge.node.name === "4-sex-shakespeare-mobile"
+  )
+  const displayGuideline4 = isPhoneWide
+    ? sexShakespeareMobile
+    : sexShakespeareDeskTop
+
+  const shakespeare = data.allFile.edges.find(
+    edge => edge.node.name === "shakespeare"
+  )
+
+  const feelYourselfDesktop = data.allFile.edges.find(
+    edge => edge.node.name === "5-feel-yourself"
+  )
+
+  const feelYourselfMobile = data.allFile.edges.find(
+    edge => edge.node.name === "5-feel-yourself-mobile"
+  )
+
+  const displayGuideline5 = isPhoneWide
+    ? feelYourselfMobile
+    : feelYourselfDesktop
+
+  const bathtub = data.allFile.edges.find(edge => edge.node.name === "bathtub")
+
+  const haveFunDesktop = data.allFile.edges.find(
+    edge => edge.node.name === "6-have-fun-and-keep-going"
+  )
+  const haveFunMobile = data.allFile.edges.find(
+    edge => edge.node.name === "6-have-fun-and-keep-going-mobile"
+  )
+
+  const displayGuideline6 = isPhoneWide ? haveFunMobile : haveFunDesktop
+
+  const zoom = data.allFile.edges.find(edge => edge.node.name === "zoom")
+
+  const madLibsDesktop = data.allFile.edges.find(
+    edge => edge.node.name === "7-fucking-mad-libs"
+  )
+
+  const madLibsMobile = data.allFile.edges.find(
+    edge => edge.node.name === "madlib"
+  )
+
+  const displayMadibs = isPhoneWide ? madLibsMobile : madLibsDesktop
+
+  const cloudDesktop = data.allFile.edges.find(
+    edge => edge.node.name === "cloud"
+  )
+  const cloudMobile = data.allFile.edges.find(
+    edge => edge.node.name === "cloud-mobile"
+  )
+
+  const displayCloud = isPhoneWide ? cloudMobile : cloudDesktop
+
+  const aboutKim = data.allFile.edges.find(
+    edge => edge.node.name === "about-kim"
+  )
+
+  const layer21 = data.allFile.edges.find(edge => edge.node.name === "layer-21")
```

Comment:
Not needed, since we can use the `<LineBreak>` component. Remember to remove the `layer-21` file from the `images` folder as well